postreise/plot/plot_upgrades_map.py

```diff
@@ -7,41 +7,42 @@
     get_dcline_differences,
 )
 from postreise.plot import colors
-from postreise.plot.plot_states import get_state_borders
-from postreise.plot.projection_helpers import project_borders, project_branch
+from postreise.plot.plot_states import plot_states
+from postreise.plot.projection_helpers import project_branch
 
 
 def _map_upgrades(
     branch_merge,
     dc_merge,
-    state_shapes,
-    b2b_indices,
+    b2b_indices=None,
     diff_threshold=100,
-    all_branch_scale=1e-3,
-    diff_branch_scale=1e-3,
+    all_branch_scale=1,
+    diff_branch_scale=0.5,
     all_branch_min=0.1,
     diff_branch_min=1.0,
     b2b_scale=5e-3,
```

Comment:
Is that possible to have the factor around 1 (as we discussed previously)

Reply:
They will be 1 by default (1 pixel/GW).

Reply:
Not sure I understand. The default value is `5e-3`

Reply:
Ah, I see which line you're referring to now. Yes, I'll make that change. The new factor will be `5`.

Reply:
Done.